H/HDF5/build_tarballs.jl

```diff
@@ -387,48 +255,39 @@ platforms = filter(p -> !(p["mpi"] == "mpitrampoline" && Sys.isfreebsd(p)), plat
 products = [
-    # Since we use the msys binaries for Windows, we can only define
-    # those products that are provided by msys as well. These are
-    # just the regular and the high-level libraries.
-
     # # HDF5 tools
-    # ExecutableProduct("h5clear", :h5clear),
-    # ExecutableProduct("h5copy", :h5copy),
-    # ExecutableProduct("h5debug", :h5debug),
-    # ExecutableProduct("h5delete", :h5delete),
-    # ExecutableProduct("h5diff", :h5diff),
-    # ExecutableProduct("h5dump", :h5dump),
-    # ExecutableProduct("h5format_convert", :h5format_convert),
-    # ExecutableProduct("h5import", :h5import),
-    # ExecutableProduct("h5jam",:h5jam),
-    # ExecutableProduct("h5ls", :h5ls),
-    # ExecutableProduct("h5mkgrp", :h5mkgrp),
-    # ExecutableProduct("h5perf_serial",:h5perf_serial),
-    # ExecutableProduct("h5repack", :h5repack),
-    # ExecutableProduct("h5repart", :h5repart),
-    # ExecutableProduct("h5stat", :h5stat),
-    # ExecutableProduct("h5unjam", :h5unjam),
-    # ExecutableProduct("h5watch", :h5watch),
+    ExecutableProduct("h5clear", :h5clear),
+    ExecutableProduct("h5copy", :h5copy),
+    ExecutableProduct("h5debug", :h5debug),
+    ExecutableProduct("h5delete", :h5delete),
+    ExecutableProduct("h5diff", :h5diff),
+    ExecutableProduct("h5dump", :h5dump),
+    ExecutableProduct("h5format_convert", :h5format_convert),
+    ExecutableProduct("h5import", :h5import),
+    ExecutableProduct("h5jam",:h5jam),
+    ExecutableProduct("h5ls", :h5ls),
+    ExecutableProduct("h5mkgrp", :h5mkgrp),
+    ExecutableProduct("h5perf_serial",:h5perf_serial),
+    ExecutableProduct("h5repack", :h5repack),
+    ExecutableProduct("h5repart", :h5repart),
+    ExecutableProduct("h5stat", :h5stat),
+    ExecutableProduct("h5unjam", :h5unjam),
+    ExecutableProduct("h5watch", :h5watch),
 
     # HDF5 libraries
     LibraryProduct("libhdf5", :libhdf5),
-    # LibraryProduct("libhdf5_cpp", :libhdf5_cpp),
-    # LibraryProduct("libhdf5_fortran", :libhdf5_fortran),
+    LibraryProduct("libhdf5_cpp", :libhdf5_cpp),
+    LibraryProduct("libhdf5_fortran", :libhdf5_fortran),
     LibraryProduct("libhdf5_hl", :libhdf5_hl),
-    # LibraryProduct("libhdf5_hl_cpp", :libhdf5_hl_cpp),
-    # LibraryProduct("libhdf5hl_fortran", :libhdf5_hl_fortran),
+    LibraryProduct("libhdf5_hl_cpp", :libhdf5_hl_cpp),
+    LibraryProduct("libhdf5hl_fortran", :libhdf5_hl_fortran),
 ]
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    # For OpenMP we use libomp from `LLVMOpenMP_jll` where we use LLVM as compiler (BSD 
-    # systems), and libgomp from `CompilerSupportLibraries_jll` everywhere else. 
-    Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae");
-               platforms=filter(!Sys.isbsd, platforms)),
-    Dependency(PackageSpec(name="LLVMOpenMP_jll", uuid="1d63c593-3942-5779-bab2-d838dc0a180e");
-               platforms=filter(Sys.isbsd, platforms)),
```

Comment:
@giordano OpenMP was removed as a dependency.

Reply:
Ah, I didn't see that addressed in the comments after my comment, I missed it had been changed before! The diff was quite large :smiling_face_with_tear: